htp/htp_util.c

```diff
@@ -2536,3 +2536,54 @@ htp_uri_t *htp_uri_alloc(void) {
 char *htp_get_version(void) {
     return HTP_VERSION_STRING_FULL;
 }
+
+htp_status_t htp_header_has_token(const unsigned char *hvp, size_t hvlen, const unsigned char *value) {
+    int state = 0;
+    // offset to compare in value
+    size_t v_off = 0;
+    for (size_t i = 0; i < hvlen; i++) {
+        switch (state) {
+            case 0:
+                if (hvp[i] == 0) {
+                    // skip 0 char
+                    continue;
+                }
+                if (v_off == 0 && hvp[i] == ' ') {
```

Comment:
should we use htp_is_space() or similar to be more liberal about spaces?

Reply:
Looks a good idea